lib/index.js

```diff
@@ -1221,7 +1265,12 @@ Stream.prototype.pull = function (f) {
  * @id write
  * @section Stream Objects
  * @name Stream.write(x)
- * @param x - the value to write to the Stream
+ * @param {any} x - the value to write to the Stream
+ * @param {String} encoding - (optional) Ignored. This parameter is only here to
+ *      match the Writable signature.
```

Comment:
(suggested edit -- see below): If it is callable, it is taken as the callback argument.